orb_models/forcefield/pretrained.py

```diff
@@ -110,12 +128,57 @@ def orb_v1(
     return model
 
 
+def orb_v2(
+    weights_path: str = "https://storage.googleapis.com/orbitalmaterials-public-models/forcefields/orbff-v2-20240930.ckpt",  # noqa: E501
+    device: Union[torch.device, str] = None,
+):
+    """Load ORB v1."""
+    base = get_base(
+        num_edge_in_features=23,
+        distance_cutoff=True,
+        attention_gate="sigmoid",
+        rbf_transform="gaussian",
+    )
+
+    model = GraphRegressor(
+        graph_head=EnergyHead(
+            latent_dim=256,
+            num_mlp_layers=1,
+            mlp_hidden_dim=256,
+            target="energy",
+            node_aggregation="mean",
+            reference_energy_name="vasp-shifted",
+            train_reference=True,
+            predict_atom_avg=True,
+        ),
+        node_head=NodeHead(
+            latent_dim=256,
+            num_mlp_layers=1,
+            mlp_hidden_dim=256,
+            target="forces",
+            remove_mean=True,
+        ),
+        stress_head=GraphHead(
+            latent_dim=256,
+            num_mlp_layers=1,
+            mlp_hidden_dim=256,
+            target="stress",
+            compute_stress=True,
+        ),
+        model=base,
+    )
+
+    model = load_model_for_inference(model, weights_path, device)
+
+    return model
+
+
 def orb_d3_v1(
     weights_path: str = "https://storage.googleapis.com/orbitalmaterials-public-models/forcefields/orb-d3-v1-20240902.ckpt",
     device: Union[torch.device, str] = None,
 ):
     """ORB v1 with D3 corrections."""
-    base = get_base()
+    base = get_base(attention_gate="sigmoid")
```

Comment:
I don't think V1 used the sigmoid attention?

Reply:
yeah sorry that was a typo!